samples/kubernetes/end2end/docs/05-prometheus.md

```diff
@@ -23,26 +24,30 @@ kubectl -n monitoring get pod -l app=prometheus
 > output
 ```
 NAME                                            READY   STATUS    RESTARTS   AGE
-prometheus-kube-state-metrics-dbfff5557-jc5sp   1/1     Running   0          25h
-prometheus-node-exporter-9t8mp                  1/1     Running   0          25h
-prometheus-server-5789fffc86-qw4mw              2/2     Running   0          25h
+prometheus-alertmanager-7456cdb5b8-9vfmd        2/2     Running   0          1h
+prometheus-kube-state-metrics-dbfff5557-f22xx   1/1     Running   0          1h
+prometheus-node-exporter-6n7rj                  1/1     Running   0          1h
+prometheus-server-5789fffc86-mmh8f              2/2     Running   0          1h
 ```
 Check the Prometheus sevices.
 ```
 kubectl -n monitoring get svc -l app=prometheus
 ```
 > output
 ```
-NAME                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
-prometheus-kube-state-metrics   ClusterIP   None            <none>        80/TCP         25h
-prometheus-node-exporter        ClusterIP   None            <none>        9100/TCP       25h
-prometheus-server               NodePort    10.99.199.236   <none>        80:30000/TCP   25h
-
+NAME                            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)        AGE
+prometheus-alertmanager         NodePort    10.108.31.234    <none>        80:32000/TCP   1h
+prometheus-kube-state-metrics   ClusterIP   None             <none>        80/TCP         1h
+prometheus-node-exporter        ClusterIP   None             <none>        9100/TCP       1h
+prometheus-server               NodePort    10.103.186.222   <none>        80:30000/TCP   1h
 ```
+
 ### Access the Prometheus web UI
 Now you can access the Prometheus web UI in your browser with the URL `http://<HostIP>:30000`.  
 In the top menu, click `Status` and then `Targets`. The target page is displayed. Go to the bottom of the page. You'll find that the two endpoints of `wls-domain1` are up and healthy.
```

Comment:
Go to the bottom of the page, you'll find that the two endpoints of `wls-domain1` are up and healthy.

Reply:
Just add a comma after Go to the bottom of the page.  It reads better.